models/database.go

```diff
@@ -695,6 +695,14 @@ var databaseSchema = [][]string{
 		`ALTER TABLE percona_sso_details
 			ADD COLUMN pmm_server_name VARCHAR`,
 	},
+	60: {
+		`ALTER TABLE percona_sso_details
+			DROP COLUMN client_id, 
+			DROP COLUMN client_secret,
```

Comment:
shouldn't we migrate value from client_id into pmm_managed_client_id? and the same for secret?

Reply:
done @BupycHuk  , changed it to RENAME